lib/pull-request/merge-back.ts

```diff
@@ -88,7 +83,7 @@ interface CommonMergeOptions extends pr.AutoPullRequestOptions {
   condition?: string;
 }
 
-export interface AutoMergeBackOptions extends CommonMergeOptions {
+export interface AutoMergeBackOptionsWithStage extends AutoMergeBackOptions {
```

Comment:
Would StageMergeBackOptions or AutoMergeBackStageOptions. I prefer the former.

Reply:
`StageMergeBackOptions` sounds like we're merging back from a stage. `AutoMergeBackStageOptions` could be ok, I guess. How about `AutoMergeBackPipelineOptions`, so if we add something else besides a stage in the future, we don't have to rename again?